src/Feature/Catalog/client/Integration/api/ProductSearch/api.ts

```diff
@@ -14,24 +14,60 @@
 
 import axios from 'axios';
 
+import { ProductsSearchRequest, SortDirection } from 'Feature/Catalog/client/dataModel.Generated';
```

Comment:
Usually we split imports on FE
libraries -> foundation -> feature -> current project imports -> styles import

Reply:
this behavior is configured via StyleCop settings